src/optimization/problem.cpp

```diff
@@ -252,6 +253,21 @@ ExitStatus Problem::solve(const Options& options, [[maybe_unused]] bool spy) {
     }
 #endif
 
+    const auto [bound_constraint_mask, bounds, conflicting_bound_indices] =
+        get_bounds(m_decision_variables, m_inequality_constraints, A_i.value());
+    if (!conflicting_bound_indices.empty()) {
+#ifndef SLEIPNIR_DISABLE_DIAGNOSTICS
```

Comment:
You could probably get away without this #ifndef since the print function gets #define'd away when the macro is set.